README.md

```diff
@@ -7,7 +7,14 @@
 [![devDependency Status](https://david-dm.org/hoodiehq/hoodie-app-tracker/dev-status.svg)](https://david-dm.org/hoodiehq/hoodie-app-tracker#info=devDependencies)
 
 Tracker is a simple Hoodie app, meant to be a starting point to play and build
-with Hoodie.
+with Hoodie. Find the repository on [GitHub](https://github.com/hoodiehq/hoodie-app-tracker).
+
+## Requirements
+
+Tracker needs `node >=4`. To see which version you have installed, run
```

Comment:
I think it's okay to say Tracker here as we're talking about the app. We know it's because of Hoodie but I'm okay with saying "To run the tracker, you need v4 or above"